aspnet/conceptual-overview/aspnet.rst

```diff
@@ -104,8 +124,23 @@ ASP.NET Core uses a new configuration model for handling of simple name-value pa
 
 See :doc:`/fundamentals/configuration` for more details on the new configuration system and :doc:`/fundamentals/environments` for details on how to work with environments in ASP.NET Core.
 
+Build web UI and web APIs using MVC
+-----------------------------------
+
+- ASP.NET Core helps you create well-factored and testable web apps. See :doc:`/mvc/index` and :doc:`/testing/index`.
+- `Razor <http://www.asp.net/web-pages/overview/getting-started/introducing-razor-syntax-c>`__ provides a productive language to create :doc:`Views </mvc/views/index>`
```

Comment:
We really need to get the [Razor](https://github.com/aspnet/Docs/issues/130) topic in place